src/oaklib/implementations/simpleobo/simple_obo_implementation.py

```diff
@@ -912,9 +912,8 @@ def _tv_dict(stanza: Stanza) -> Dict[str, List[str]]:
                         id=_id(), about_node=t1id, new_value=vals2list[0], old_value=None
                     )
                 else:
-                    #! KGCL may have NameDeletion in the future.
-                    yield kgcl.NodeRename(
-                        id=_id(), about_node=t2id, old_value=vals2list[0], new_value=None
+                    yield kgcl.NodeDeletion(
+                        id=_id(), about_node=t1id, old_value=vals1list[0], new_value=None
```

Comment:
@cmungall : This is the only real change in the PR. I ran `tox -e format` that changed al other files. 

Reason: GH Actions were failing and complaining about imports not being sorted.